### DISCUSSION TITLE
How do I delete an user?

### DISCUSSION THREAD

**Discussion by erkexzcx** [hay-kot/homebox#868] [Q&A]
Title. I can't seem to find any documentation on how users are being deleted or managed (manually? by an admin account?). Any advise?

**Comment by hay-kot** [comment]
There is no user management mechanism in Homebox. 

Each user that signs up get's their own collection of assets, locations, and labels unless they are invited using a link (you can find that link your profile page) 

Currently, I don't see user management as a core feature for the Home User. I think any work in this area would need to come from the community or be sponsored work for me.
